requirements-base.txt

```diff
@@ -3,6 +3,7 @@
 beautifulsoup4>=4.7.1
 boto3>=1.34.128
 botocore>=1.34.8
+brotli>=1.1.0
```

Comment:
This is just making it explicit as we use it for compression in the new SDK. It is optional but highly valuable.